modules/launcher/src/main/java/org/mule/runtime/module/launcher/DefaultMuleContainer.java

```diff
@@ -113,7 +120,10 @@ public class DefaultMuleContainer implements MuleContainer {
       } else {
         log4jContextFactory = createAndInstall();
       }
-      configureSelector(log4jContextFactory);
+
+      if (getBoolean(SINGLE_APP_MODE_PROPERTY)) {
```

Comment:
why is this tied to `MULE_SIMPLE_LOG`?

Reply:
it is tied to not "Simple log". I'll change it anyway.